packages/react-theme-provider/src/getTokens.ts

```diff
@@ -59,6 +59,61 @@ export function getTokens(theme: Theme): Tokens {
         },
       },
 
+      // ghost is currently only mapped for primary button to use.
```

Comment:
you meant button, right?
```suggestion
      // ghost is currently only mapped for button to use.
```